dev-docs/bidders/publir.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: bidder
+title: Publir
+description: Prebid Publir Bidder Adapter
+multiformat_supported: will-bid-on-any
+pbjs: true
+biddercode: publir
+media_types: banner
+schain_supported: true
+gdpr_supported: true
+usp_supported: true
+floors_supported: true
+userIds: all
+gvl_id: none
```

Comment:
Please add `safeframes_ok: true/false`. It helps vetting your adapter

Reply:
Thank you, Have added in the commit